subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -2407,6 +2407,15 @@ static inline int rx_demux_rx(memq_link_t *link, struct node_rx_hdr *rx)
 		ull_scan_aux_release(link, rx);
 	}
 	break;
+#if defined(CONFIG_BT_CTLR_SYNC_PERIODIC)
+	case NODE_RX_TYPE_SYNC:
+
+	{
+		(void)memq_dequeue(memq_ull_rx.tail, &memq_ull_rx.head, NULL);
+		ull_sync_established_report(link, rx);
+	}
+	break;
```

Comment:
```suggestion
	case NODE_RX_TYPE_SYNC:
		(void)memq_dequeue(memq_ull_rx.tail, &memq_ull_rx.head, NULL);
		ull_sync_established_report(link, rx);
		break;
```

Reply:
I've just removed empty line after `case NODE_RX_TYPE_SYNC` to follow convention in other case statements in the function.

Reply:
That's fine; didn't see that the other cases used brackets as well.